packager/VisibilityChecker.cc

```diff
@@ -496,31 +512,140 @@ class VisibilityCheckerPass final {
                                             std::vector<ast::ParsedFile> files) {
         Timer timeit(gs.tracer(), "visibility_checker.check_visibility");
         auto taskq = std::make_shared<ConcurrentBoundedQueue<size_t>>(files.size());
-        absl::BlockingCounter barrier(std::max(workers.size(), 1));
 
+        // [(package, packageToImport, isTestImport)]
+        auto toImportQ = std::make_shared<BlockingBoundedQueue<
+            std::vector<std::tuple<core::packages::MangledName, core::packages::MangledName, bool>>>>(files.size());
+        // [(package, packageToImport, Loc)]
+        auto replaceTestImportQ = std::make_shared<BlockingBoundedQueue<
+            std::vector<std::tuple<core::packages::MangledName, core::packages::MangledName, core::Loc>>>>(
+            files.size());
         for (size_t i = 0; i < files.size(); ++i) {
             taskq->push(i, 1);
         }
 
-        workers.multiplexJob("VisibilityChecker", [&gs, &files, &barrier, taskq]() {
+        workers.multiplexJob("VisibilityChecker", [&gs, &files, taskq, toImportQ, replaceTestImportQ]() {
             size_t idx;
+            int processedByThread = 0;
+            std::vector<std::tuple<core::packages::MangledName, core::packages::MangledName, bool>> importCapacitor;
```

Comment:
What is a "capacitor" in this context? I've only ever heard this used in [electrical engineering](https://en.wikipedia.org/wiki/Capacitor)